cmd/ro/main.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/cloudflare/redoctober/client"
+	"github.com/cloudflare/redoctober/core"
+
+	"github.com/lziest/gopass"
+)
+
+var action, user, pswd, userEnv, pswdEnv, server, caPath string
+
+var owners, lefters, righters, inPath, labels, outPath, outEnv string
+
+var uses int
+
+var time, users string
+
+func registerFlags() {
+	flag.StringVar(&action, "action", "", "client action")
+	flag.StringVar(&server, "server", "localhost:8080", "server address")
+	flag.StringVar(&caPath, "ca", "", "ca file path")
+	flag.StringVar(&owners, "owners", "", "comma separated owner list")
+	flag.StringVar(&users, "users", "", "comma separated user list")
+	flag.IntVar(&uses, "uses", 0, "number of delegated key uses")
+	flag.StringVar(&time, "time", "0h", "duration of delegated key uses")
+	flag.StringVar(&lefters, "left", "", "comma separated left owners")
+	flag.StringVar(&righters, "right", "", "comma separated right owners")
+	flag.StringVar(&labels, "labels", "", "comma separated labels")
+	flag.StringVar(&inPath, "in", "", "input data file")
+	flag.StringVar(&outPath, "out", "", "output data file")
+	flag.StringVar(&outEnv, "outenv", "", "env variable for output data")
+	flag.StringVar(&user, "user", "", "username")
+	flag.StringVar(&pswd, "password", "", "password")
+	flag.StringVar(&userEnv, "userenv", "RO_USER", "env variable for user name")
+	flag.StringVar(&pswdEnv, "pswdenv", "RO_PASS", "env variable for user password")
+}
+
+func getUserCredentials() {
+	user = os.Getenv(userEnv)
+	pswd = os.Getenv(pswdEnv)
+	if user == "" || pswd == "" {
+		fmt.Print("Username:")
+		fmt.Scanf("%s", &user)
+		var err error
+		pswd, err = gopass.GetPass("Password:")
+		processError(err)
+	}
+}
+
+func processError(err error) {
+	if err != nil {
+		fmt.Println("error:", err)
```

Comment:
This should probably print to `stderr`.
`fmt.Fprintln(os.Stderr,...)`
Or just call `log.Fatal("error:",err)`